### PR TITLE
chore: remove a few unused deps

### PR DESCRIPTION
**Pull request by JustinBeckwith**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -79,7 +79,6 @@
     "gcp-metadata": "^0.9.0",
     "google-auth-library": "^2.0.0",
     "google-gax": "^0.20.0",
-    "google-proto-files": "^0.17.0",
```
Comment (ofrobots) [inline package.json]: Is this related at all to #133?

Comment (JustinBeckwith) [reply package.json]: It isn't (that I know of). This package is used by gax, but just kinda hanging out here as far as I can tell.
